physionet-django/user/models.py

```diff
@@ -685,3 +685,13 @@ def file_root(self):
 
     def get_full_name(self):
         return ' '.join([self.first_names, self.last_name])
+
+    def get_latest_by_user(self):
+        return CredentialApplication.objects.filter(user=self.user).last()
+
+    def is_last_by_user(self):
```

Comment:
"latest" is clearer than "last", so perhaps also use that here (e.g. `def is_latest_by_user`)